cmd/gcp-controller-manager/app/gke_approver.go

```diff
@@ -73,6 +73,8 @@ const (
 	// TODO(awly): eventually this should be true. But that makes testing
 	// harder, so disabling or now.
 	validateClusterMembership = false
+
+	metricLabelNone = "none"
```

Comment:
are labels ordered? if not, this label should be more descriptive.

Reply:
You mean "status" and "kind"? They are always ordered.
Changed to "not_approved", but it should be understandable either way